recipes/mbsolve/bld.bat

```diff
@@ -0,0 +1,15 @@
+mkdir build
+cd build
+
+set "CXXFLAGS="
+
+cmake ^
+    -G "Ninja" ^
+    -DCMAKE_PREFIX_PATH="%LIBRARY_PREFIX%" ^
+    -DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%" ^
+    -DPYTHON_LIBRARY="%PREFIX%\libs\python%CONDA_PY%.lib" ^
+    -DCMAKE_BUILD_TYPE=Release ^
+    ..
+
+cmake --build . --config Release
+cmake --build . --config Release --target install
```

Comment:
This file needs exit clauses after each line

Reply:
I am not too familiar with Windows batch scripts -- not exactly sure what you mean. Could you point me to an example (recipe), please?

Reply:
I am not too familiar with Windows batch scripts -- not exactly sure what you mean. Could you point me to an example (recipe), please?

Reply:
See this example: https://github.com/conda-forge/make-feedstock/blob/master/recipe/bld.bat#L5

Reply:
Thanks! I have added another commit that addresses the missing clauses.